src/main/java/ch/wisv/areafiftylan/controller/TicketTransferRestController.java

```diff
@@ -44,6 +50,15 @@ public TicketTransferRestController(TicketService ticketService) {
         return createResponseEntity(HttpStatus.OK, "Ticket transfer successfully cancelled");
     }
 
+    @PreAuthorize("isAuthenticated() and @currentUserServiceImpl.hasAnyTicket(principal)")
```

Comment:
Why is the "hasAnyTicket" a security/authorization check? A user may still query right, but would just get an empty set as a result.